packages/scaffold/src/views/w3m-wallet-send-preview-view/index.ts

```diff
@@ -114,11 +137,27 @@ export class W3mWalletSendPreviewView extends LitElement {
     return null
   }
 
-  private onSendClick() {
-    RouterController.reset('Account')
-    setTimeout(() => {
-      SendController.resetSend()
-    }, 200)
+  onSendClick() {
+    if (this.token?.address && this.sendTokenAmount && this.receiverAddress) {
+      SendController.sendERC20Token({
```

Comment:
I'd prefer to move this logic to `SendController` and on the UI, just call `SendController.sendToken()` which will do conditions inside that function. This would decrease the business logic on the UI side and move the maintainability to controllers (if we have all these states in `SendController`) - We could go this was as well, not a strong opinion.